002/exercise/readme.md

```diff
@@ -6,9 +6,16 @@
 - Learn how to optimize a model through Gradient Descent
 
 # Task
-- Follow the Jupyter Notebook and complete the required tasks:
+- Follow the Jupyter Notebook  and complete the required tasks:
```

Comment:
This may be breaking some open source rules of engagement around PR comments, but this extra space is unnecessary

```suggestion
- Follow the Jupyter Notebook and complete the required tasks:
```